scripts/install_dependencies/install_openvino_dependencies.sh

```diff
@@ -163,6 +166,7 @@ elif [ "$os" == "centos7" ] || [ "$os" == "centos8" ] ||
     fi
 
     pkgs_dev=(gcc gcc-c++ make glibc libstdc++ libgcc cmake3 "json-devel.$arch" "zlib-devel.$arch" sudo)
+    pkgs_gpu=()
```

Comment:
why is it empty?
For Ubuntu we need `ocl-icd-libopencl1`, some packages are also available for other platforms

Reply:
As I said in PR #15160 for ubu ocl-icd-libopencl1 listed as dependency by intel-opencl-icd